src/app/routes/cpsAsset/getInitialData/index.js

```diff
@@ -60,7 +61,24 @@ const transformJson = async (json, pathname) => {
   }
 };
 
-export default async ({ path: pathname, service, variant, pageType }) => {
+// Moved to a different function to additional data processing other than most watched
+const processAdditionalData = ({ data, service, path: pathname, toggles }) => {
+  const processedData = processMostWatched({
+    data,
+    service,
+    path: pathname,
+    toggles,
+  });
+
+  return processedData;
+};
```

Comment:
Is this function for some future work?

All it seems to be doing is calling another function with the exact same arguments and return value

It seems equivalent to:

```suggestion
const processAdditionalData = processMostWatched;
```

Reply:
Yeah, for future work. I'll take it out. When we get to the point where we need it we can add it back